whipper/test/common.py

```diff
@@ -70,8 +70,8 @@ def readCue(self, name):
         ret = open(os.path.join(os.path.dirname(__file__), name)).read(
             ).decode('utf-8')
         ret = re.sub(
-            'REM COMMENT "Morituri.*',
-            'REM COMMENT "Morituri %s"' % (morituri.__version__),
+            'REM COMMENT "Whipper.*',
+            'REM COMMENT "Whipper %s"' % (whipper.__version__),
             ret, re.MULTILINE)
```

Comment:
I think the original code is broken here. All the cue sheets used in the tests have 'morituri' in all lower case (and so does the code that generates those lines). Since the regular expression doesn't ignore case, that means the substitution currently has no effect.

This doesn't have to be fixed in this PR, but it should be fixed (by making it "whipper" rather than "Whipper").